psi4/driver/procrouting/interface_cfour.py

```diff
@@ -389,8 +389,9 @@ def run_cfour(name, **kwargs):
 
     # new skeleton wavefunction w/mol, highest-SCF basis (just to choose one), & not energy
     #   Feb 2017 hack. Could get proper basis in skel wfn even if not through p4 basis kw
+    #   May 2017 hack: [0] to skip ecp
     gobas = core.get_global_option('BASIS') if core.get_global_option('BASIS') else 'sto-3g'
-    basis = core.BasisSet.build(molecule, "ORBITAL", gobas)
+    basis = core.BasisSet.build(molecule, "ORBITAL", gobas)[0]
```

Comment:
I think you need to remove this again after Andy's change? You might also want to throw an error if `basis.has_ECP()` is `true`.